scss/components/_card.scss

```diff
@@ -83,6 +84,10 @@ $card-margin: $global-margin !default;
     flex: 0 1 auto;
   }
 
+  border-width: 0 1px 0 0;
+  border-style: solid;
+  border-color: $white;
```

Comment:
You are putting a white border on the right of every `.card-divider`, regardless on the configuration.

It works ~well in your example, because you have two cards side by side on a black background:
![unspecified](https://cloud.githubusercontent.com/assets/9939075/21699402/dc24412a-d39b-11e6-8f4e-ffdd441de19e.png)
(Even if by looking closer at your example, we can also see an unexpected white border on the right of the second card).

But for a single card on a white background, it gives:
![screen shot 2017-01-05 at 23 08 06](https://cloud.githubusercontent.com/assets/9939075/21699421/f244f56c-d39b-11e6-962d-a419bf2792c4.png)

Globally, it is not a good practice to apply border of components on their _inner elements_. Card border should be on card, and if you want to avoid "double border" when multiple cards are side-to-side, simply disable `border-right` except for the last card.